### PR TITLE
IAP-5631 API Portal T4 pagebuilder localization docs

### PR DESCRIPTION
**Pull request by yasen-petrov**
API Portal localization for T4 Page Builder (New UI) as a stand-alone page

## Deploy preview link

http://pr-2516.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_administration/apiportal_admin/localize_language_t4/

### CODE REVIEW

File: content/en/docs/apim_administration/apiportal_admin/localize_language_t4.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    * **Language files context**: Select whether the uploaded language files are for the **Site** or **Administrator** part of API Portal.
+    * **API Portal component language file**: The main language file for API Portal component.
+    * **Overridden language file**: Some texts in API Portal, such as the messages on the login pages, are stored in different filed, and you must change and upload them separately.
+    * **`*.sys.ini` language file**: This options is shown only when **Administrator** is selected as a language files context. This language file contains translations for the administrator components menu drop-down, menu, and component parameters.
+3. Click **Save & Close**.
+
+To translate labels or values of custom properties, you must add a new row to the translation `*.ini` file, with the desired `key/value` pair, and ensure that the value of the key is in capital letters. For example, to translate a custom property with label "Environment" to French, add the following line to the translation file: `ENVIRONMENT="Environnement"`.
+
+### Rename API Portal menu items
```
Comment (andreamussap) [inline content/en/docs/apim_administration/apiportal_admin/localize_language_t4.md:48]: Look at the deploy preview on this section. The list of steps is broken

**Review by dginevaAxway — APPROVED** [review]
Contented seems ok.

**Comment by yvelikova** [comment]
alabala

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | f2832bb |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2516.axway-open-docs.opendocs-builder.pcloud.axway.int |

**Review by yvelikova — APPROVED** [review]

**Pull request merged**